src/views/components/response_components/survey_summary.vue

```diff
@@ -120,13 +131,21 @@ export default {
                         ) / 10;
                     return el;
                 });
+            //show component if data exist
```

Comment:
Sorry I missed it before. You should always use `parseInt(key, 10)` to avoid accidentally parsing octal or hex.

Reply:
This could also be written more cleanly.
```
let sum = Math.sum(...el.responses.map(x => parseInt(x, 10)))
el.numerical_average = Math.round(10 * sum / el.responses.length) / 10
```

Reply:
I didn't find that JS has a Math.sum function, I do see it from a math.js library would you wanna import that library to do this?

Reply:
I forgot to mention that I updated the PR last night, the `parseInt(key, 10)` issue is fixed.